modules/Rock.cmake

```diff
@@ -469,14 +469,22 @@ macro(rock_target_definition TARGET_NAME)
             list(APPEND ${TARGET_NAME}_MOC ${__moced_file})
         endforeach()
 
-        QT4_WRAP_CPP(${TARGET_NAME}_MOC_SRCS ${${TARGET_NAME}_MOC})
-        list(APPEND ${TARGET_NAME}_SOURCES ${${TARGET_NAME}_MOC_SRCS})
+	if (ROCK_QT_VERSION EQUAL 5)
+	    QT5_WRAP_CPP(${TARGET_NAME}_MOC_SRCS ${${TARGET_NAME}_MOC})
+        else()
+	    QT4_WRAP_CPP(${TARGET_NAME}_MOC_SRCS ${${TARGET_NAME}_MOC})
+	endif()
+	list(APPEND ${TARGET_NAME}_SOURCES ${${TARGET_NAME}_MOC_SRCS})
```

Comment:
Indentation looks wrong. Use 4 spaces, no tabs.

Reply:
Resolved.

Reply:
@doudou Converted all tabs to spaces throughout the whole file. Could also offer to run cmake-format beforehand to bring some better indentation to the Rock.cmake file.